sdk/python/kubeflow/metadata/metadata.py

```diff
@@ -176,6 +185,44 @@ def _flat(self, artifact):
         result[k] = v
     return result
 
+  def _get_context_id(self, reuse_workspace_if_exists):
+    ctx = self._get_existing_context()
+    if ctx is not None:
+      if reuse_workspace_if_exists:
+        return ctx.id
+      else:
+        raise ValueError('workspace name {} already exists with id {}'.format(
```

Comment:
Suggest making more clear reminder here such as set `reuse_workspace_if_exists=True` if want to reuse workspace ...